tsconfig-strict.json

```diff
@@ -427,8 +427,6 @@
     "core/templates/pages/exploration-player-page/exploration-player-page.component.spec.ts",
     "core/templates/pages/exploration-player-page/exploration-player-page.constants.ts",
     "core/templates/pages/exploration-player-page/layout-directives/correctness-footer.component.ts",
-    "core/templates/pages/exploration-player-page/layout-directives/exploration-footer.component.ts",
-    "core/templates/pages/exploration-player-page/layout-directives/exploration-footer.component.spec.ts",
```

Comment:
Can you rather fix these errors in these files?

Reply:
Thanks @vojtechjelinek. Actually, the issue is that I get ts errors in the files that are being imported by the above-mentioned two files. How should I proceed?

Reply:
How many errors? In what files? Can you fix those?

Reply:
Quite a lot actually. For eg: core/templates/pages/exploration-player-page/services/exploration-engine.service.ts. However, many files with errors have not been imported actually. For eg: core/templates/pages/exploration-player-page/services/stats-reporting.service.ts. 

https://user-images.githubusercontent.com/76519771/157934339-3e208766-1601-4787-8237-f6d6928c5a7a.mp4

Reply:
@Manan-Rathi The video seems to be corrupt and I'm not able to play it. Either post a screenshot or let me know the number of errors.

Reply:
@vojtechjelinek There are about 110 errors, 50 being just inside exploration-engine-service

Reply:
Okay, I think it is fine to remove it then.